### PR TITLE
i18n(fr) Deploy Vercel

### PR DESCRIPTION
**Pull request by agriffard**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate https://docs.astro.build/en/guides/deploy/vercel/ in French.

### CODE REVIEW

File: src/pages/fr/guides/deploy/vercel.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: Déployer votre Site Astro vers Vercel
+description: Comment déployer votre site Astro vers le web sur Vercel.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Vous pouvez utiliser [Vercel](http://vercel.com/) pour déployer un site Astro vers son global edge network avec aucune configuration.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:8]: I think this is more correct, right?
```suggestion
Vous pouvez utiliser [Vercel](http://vercel.com/) pour déployer un site Astro vers son global edge network sans aucune configuration.
```

File: src/pages/fr/guides/deploy/vercel.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+## Configuration du projet
+
+Votre projet Astro peut être déployé vers Vercel en tant que site statique, ou en tant que site rendu côté serveur (SSR).
+
+### Site Statique
+
+Votre projet Astro est un site statique par défaut. Vous n'avez pas besoin de configuration supplémentaire pour déployer un site statique Astro vers Vercel. 
+
+:::note
+Il y a actuellement un problème Vercel en affichant une page 404 sur les sites web Astro. Jusqu'à ce que ça soit corrigé, vous pouvez ajouter le fichier de configuration suivant à la racine de votre projet :
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:21]:
```suggestion
Il y a actuellement un problème Vercel affichant une page 404 sur les sites web Astro. Jusqu'à ce que ça soit corrigé, vous pouvez ajouter le fichier de configuration suivant à la racine de votre projet :
```

File: src/pages/fr/guides/deploy/vercel.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  "cleanUrls": true
+}
+```
+:::
+
+### Adapteur pour SSR
+
+Pour activer le SSR dans votre projet Astro et déployer sur Vercel:
+
+Ajoutez [l'adapteur Vercel](/fr/guides/integrations-guide/vercel/) pour activer le SSR dans votre projet Astro avec la commande `astro add`. Ceci installera l'adapteur et fera les changements appropriés dans votre fichier `astro.config.mjs`en une étape.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:34]:
```suggestion
Ajoutez [l'adapteur Vercel](/fr/guides/integrations-guide/vercel/) pour activer le SSR dans votre projet Astro avec la commande `astro add`. Ceci installera l'adapteur et fera les changements appropriés dans votre fichier `astro.config.mjs` en une étape.
```

File: src/pages/fr/guides/deploy/vercel.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+```bash
+npx astro add vercel
+```
+
+Si vous préféree installer l'adapteur manuallement à la place, complétez les deux étapes suivantes :
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:40]: Typo
```suggestion
Si vous préférez installer l'adapteur manuallement à la place, complétez les deux étapes suivantes :
```

File: src/pages/fr/guides/deploy/vercel.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+    export default defineConfig({
+      output: 'server',
+      adapter: vercel(),
+    });
+    ```
+
+## Comment déployer
+
+Vous pouvez déployer vers Vercel via lIU du site web ou utiliser l'ILC (interface en ligne de commande) de Vercel. Le processus est le même pour les sites statiques et SSR Astro.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:62]: Is "IU" (for "Interface Utilisateur") common in french? I've never heard it personally, is it a typo? In french, using "Interface" is enough IMO.
```suggestion
Vous pouvez déployer vers Vercel via l'interface du site web ou utiliser l'ILC (interface en ligne de commande) de Vercel. Le processus est le même pour les sites statiques et SSR Astro.
```

File: src/pages/fr/guides/deploy/vercel.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+### Déploiement via l'IU du site web
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:64]: Feel free to deny this one in pair with the previous one.
```suggestion
### Déploiement via l'interface utilisateur du site web
```

File: src/pages/fr/guides/deploy/vercel.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+1. Installez l'[ILC Vercel](https://vercel.com/cli) et exécutez `vercel` pour déployer.
+
+    ```bash
+    npm install -g vercel
+    vercel
+    ```
+
+2. Vercel détectera automatiquement Astro et configurera les bons paramètres.
+3. Lorsqu'il sera demandé `Want to override the settings? [y/N]`, sélectionner `N`.
+4. Votre application est déploée ! (ex : [astro.vercel.app](https://astro.vercel.app/))
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy/vercel.md:87]:
```suggestion
4. Votre application est déployée ! (ex : [astro.vercel.app](https://astro.vercel.app/))
```

**Review by MoustaphaDev — APPROVED** [review]
LGTM!

**Review by delucis — APPROVED** [review]
Thanks @agriffard & @MoustaphaDev!

**Pull request merged**